Fixes upstream BRLTTY errors (wformat-security).

This warning appears inconsistent across versions. gettext
internationalizes the string literal, but is unused in Chrome OS.

diff --git a/Programs/config.c b/Programs/config.c
index d99d93d95..2773ab852 100644
--- a/Programs/config.c
+++ b/Programs/config.c
@@ -1893,7 +1893,7 @@ restartBrailleDriver (void) {
   awaitActivityStopped(brailleDriverActivity);
   brl.hasFailed = 0;
 
-  logMessage(LOG_INFO, gettext("reinitializing braille driver"));
+  logMessage(LOG_INFO, "reinitializing braille driver");
   enableBrailleDriver();
 }
 
@@ -2406,7 +2406,7 @@ restartScreenDriver (void) {
   disableScreenDriver(gettext("screen driver restarting"));
   awaitActivityStopped(screenDriverActivity);
 
-  logMessage(LOG_INFO, gettext("reinitializing screen driver"));
+  logMessage(LOG_INFO, "reinitializing screen driver");
   enableScreenDriver();
 }
 
@@ -2850,7 +2850,7 @@ brlttyStart (void) {
   
   /* The device(s) the braille display might be connected to. */
   if (!*opt_brailleDevice) {
-    logMessage(LOG_ERR, gettext("braille device not specified"));
+    logMessage(LOG_ERR, "braille device not specified");
     return PROG_EXIT_SYNTAX;
   }
 
diff --git a/Programs/program.c b/Programs/program.c
index 548f284f9..e5af95e6f 100644
--- a/Programs/program.c
+++ b/Programs/program.c
@@ -145,7 +145,7 @@ fixInstallPath (char **path) {
     if ((programDirectory = getPathDirectory(programPath))) {
       registerProgramMemory("program-directory", &programDirectory);
     } else {
-      logMessage(LOG_WARNING, gettext("cannot determine program directory"));
+      logMessage(LOG_WARNING, "cannot determine program directory");
       programDirectory = CURRENT_DIRECTORY_NAME;
     }
 
